themes/default/content/blog/azure-container-solutions/index.md

```diff
@@ -0,0 +1,181 @@
+---
+title: "Which Azure Container Solution is right for you?"
+date: 2021-08-31T13:51:22-07:00
+draft: true
+meta_desc: "A look at the container solutions available in Azure"
+meta_image: "azure-top-5.png"
+authors:
+    - lee-briggs
+tags:
+    - azure
+    - containers
+---
+
+Containers have emerged as one of the de facto standards for running software. When adopted with the right mindset, they can drastically improve the development lifecycle and help to close the loop between local development and running your applications in the cloud.
+
+If you're at the stage of trying to run your application in Microsoft Azure, the choices can be overwhelming. The Azure Container [product page](https://azure.microsoft.com/en-gb/product-categories/containers/) lists 7 different products on their landing page, and for new users it can often be difficult to decide which of the myriad products is right for their use case. What can make it even more confusing is that often these container services can be interoperable, meaning you can use one container product from another!
```

Comment:
I'm not sure "option" works with my writing style here, can you elaborate on this suggestion a little? Grammarly seems to think the original verbiage is okay

Reply:
Ah, so `option` was just a word to put there. This suggestion comes from back when I was editing standardized exams. Generally, the lack of a word in this phrase can cause a reader who might not have English as their first language or who might have trouble with English grammar to trip up and get lost in the sentence because they can't identify an antecedent. I'd say you can safely skip this suggestion if you would rather do so as the consequences are much lower here :)